src/purple-libnotify+.c

```diff
@@ -169,6 +169,12 @@ _purple_notify_plus_email(PurplePlugin *plugin, const gchar *subject, const gcha
 }
 
 static void
+_purple_notify_auth_request(PurpleAccount *account, const char *remote_user, const char *message, PurplePlugin *plugin)
+{
+	notify_plus_send_name_notification(remote_user, _("Authorization request from %s"), message, NULL, NULL);
```

Comment:
I want to keep the buddy name as the first thing, to quickly have the context. More like “%s requested authorization”.
(Seems I failed for some other events, let’s not continue.)